scripts/toolchain/Dockerfile

```diff
@@ -11,19 +11,20 @@ WORKDIR /app/hoprnet-toolchain
 RUN mkdir -p scripts/toolchain
 
 RUN apt-get update \
- && apt-get install -y \
-      bash \
-      build-essential \
-      ca-certificates \
-      curl \
-      git \
-      jq \
-      lsof \
-      python3 \
-      unzip \
-      xz-utils \
- && rm -rf /var/lib/apt/lists/* \
- && apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false
+     && apt-get install -y \
```

Comment:
Please intend the following lines differently to highlight that they are belonging to the same command.

Reply:
Wondering, is there some fmt tool for Dockerfiles?